### PR TITLE
Use the key blinding contexts (and keep them separate)

### PR DESCRIPTION
**Pull request by chris-wood**
cc @FredericJacobs 

See the corresponding change [here](https://github.com/cloudflare/pat-go/pull/18/commits/36d6705816141f305e7cba6425639b502d50bad8).

### CODE REVIEW

**Review by tfpauly — APPROVED** [review]

File: draft-privacypass-rate-limit-tokens.md [hunk 1]
```diff
@@ -1166,6 +1167,7 @@ In pseudocode, this is as follows:
 
 ~~~
 sk_blind = BKS-BlindKeyGen()
+ctx = concat(encode(2, token_type)), "ClientBlind")
 blinded_key = BKS-BlindPublicKey(pk_sign, sk_blind)
```
Comment (tfpauly) [inline draft-privacypass-rate-limit-tokens.md:1171]:
```suggestion
blinded_key = BKS-BlindPublicKey(pk_sign, sk_blind, ctx)
```

File: draft-privacypass-rate-limit-tokens.md [hunk 2]
```diff
@@ -1166,6 +1167,7 @@ In pseudocode, this is as follows:
 
 ~~~
 sk_blind = BKS-BlindKeyGen()
+ctx = concat(encode(2, token_type)), "ClientBlind")
```
Comment (FredericJacobs) [inline draft-privacypass-rate-limit-tokens.md:1170]: Is there something else we want to bind in the context?

Not thinking of anything in particular but just the token type and a string does not seem like it's binding a lot.

Maybe something to take another look at in the pats-analysis?

**Review by FredericJacobs — APPROVED** [review]
Looks good to me overall. I wonder if the binding in the context is sufficient or if other things should be added.

File: draft-privacypass-rate-limit-tokens.md (hunk 2 above)
Comment (chris-wood) [reply draft-privacypass-rate-limit-tokens.md:1170]: So I think we need to this context to say "this is client blinding for the rate limited privacy pass protocol," and I think what we have suffices for that. But I'm open to adding more if we think it's needed.

**Pull request merged**